src/System.ComponentModel.Annotations/tests/System/ComponentModel/DataAnnotations/CustomValidationAttributeTests.cs

```diff
@@ -304,5 +304,21 @@ public DerivedValidationResult(string errorMessage): base(errorMessage)
 
             public string AdditionalData { get; set; }
         }
+
+        [Fact]
+        public static void AllowMultiple()
+        {
+            Assert.Equal(3, TypeDescriptor.GetAttributes(typeof(AllowMultipleClass)).Count);
+        }
+
+        [CustomValidation(typeof(AllowMultipleClass), nameof(AllowMultipleClass.Method1))]
+        [CustomValidation(typeof(AllowMultipleClass), nameof(AllowMultipleClass.Method2))]
```

Comment:
For more coverage, you can change the last one to be Method2 again and then change the test to verify that it returns 2.